<launch>
    <!-- Inicia a simulação no Gazebo -->
    <!-- <include file="$(find pioneer3at_gazebo)/launch/pioneer3at_empty_world.launch" /> -->

    <param name="goal_tolerance" value="0.1"/>
    <param name="max_linear_speed" value="0.4"/>
    <param name="max_angular_speed" value="0.8"/>

    <!-- Carrega os parâmetros -->
    <rosparam file="$(find tangent_bug_2dnav)/config/params.yaml" />

    <!-- Inicia o nó do algoritmo Tangent Bug -->
    <node name="tangent_bug_2dnav" pkg="tangent_bug_2dnav" type="tangent_bug_2dnav" output="screen"/>
</launch>
  